src/Controller/BridgeGameController.js

```diff
@@ -0,0 +1,80 @@
+const BridgeRandomNumberGenerator = require('../BridgeRandomNumberGenerator');
+const BridgeMaker = require('../BridgeMaker');
+const BridgeGame = require('../Domain/BridgeGame');
+const BridgeMap = require('../Domain/BridgeMap');
+const Bridge = require('../Domain/Bridge');
+const OutputView = require('../View/OutputView');
+const InputView = require('../View/InputView');
+const SETTING = require('../constants/gameSetting');
+
+class BridgeGameController {
+  #bridgeMap = new BridgeMap();
+  #bridgeGame = new BridgeGame();
```

Comment:
[eslint lines-between-class-members](https://eslint.org/docs/latest/rules/lines-between-class-members)에 따르면 아래처럼 한 줄 띄어쓰는 걸 권장하는 것 같아요!
```suggestion
  #bridgeMap = new BridgeMap();
  
  #bridgeGame = new BridgeGame();
```